### ISSUE TITLE
Support  application/vnd.github.VERSION.diff

### ISSUE THREAD

**Issue by aelent-scalyr** [PyGithub/PyGithub#983]
As documented here: https://developer.github.com/v3/media/#commits-commit-comparison-and-pull-requests. 

Required for determining position for creating a review comment.

**Comment by ericwb** [comment]
Yes, this would be very nice. You can do this from the GitHub CLI using `gh pr diff <num>`.

It makes a GET request to `/api/v3/:repos:/:org:/repo/pulls/:num:`

It sets the header to:
`Accept: application/vnd.github.v3.diff`
